grpc-locations/src/main/proto/locationservice-v1.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "io.quarkus.sample.superheroes.location.grpc";
+option java_outer_classname = "LocationServiceProto";
+
+package io.quarkus.sample.superheroes.location.v1;
+
+enum LocationType {
+  PLANET = 0;
+  CITY = 1;
+}
+
+message Location {
+  string name = 1;
+  string description = 2;
+  string picture = 3;
+  LocationType type = 4;
+}
+
+message RandomLocationRequest {}
```

Comment:
You could have reused `Empty` from Google.

Reply:
I did that originally, but then as I was reading about protobuf, Empty would make any kind of future change not backwards compatible. Having an empty request/response object allows you to add attributes in the future should you need/want to.